.github/workflows/check-devbox.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install devbox
-        uses: jetpack-io/devbox-install-action@a0bd99f5de03156cc900bbbeb830f07d3d39aab0 # v0.6.1
+        uses: jetpack-io/devbox-install-action@4a7f1d5332cc72057d5e8080edebfcdf652e642e # v0.8.0
         with:
           enable-cache: true
           devbox-version: 0.5.10
```

Comment:
While we're at it, mind bumping this to `0.9.0`? Unless it causes any issues. If so, we can punt and I'll update devbox and its lock file later.